docs/_includes/generated/expert_scheduling_section.html

```diff
@@ -14,6 +14,12 @@
             <td>Boolean</td>
             <td>Enable the slot spread out allocation strategy. This strategy tries to spread out the slots evenly across all available <span markdown="span">`TaskExecutors`</span>.</td>
         </tr>
+        <tr>
+            <td><h5>slotmanager.number-of-slots.max</h5></td>
```

Comment:
Given that there's another ticket (FLINK-15959) for introducing min slots limit to Flink cluster, I think it might be good to have a pair of `xyz.[min|max]`. Of course one could also argue for a pair of `min-xyz` and`max-xyz`. I've not strong preference here.

Reply:
I also do not have a strong preference here. `max-number-of-slots` looks good to me.

Reply:
Yes, I'd like to. However, since it is a project-wide naming convention, I think it probably out of the scope of this PR. When we read a consensus in dev ML, it is another job to apply this convention to all existing configs. WDYT?

Reply:
Yes, that's what I meant. It does not make sense to wait with this PR until we reach consensus.